tests/base/src/test/resources/org/wildfly/security/ssl/wildfly-ssl-test-config-v1_5.xml

```diff
@@ -36,11 +36,42 @@
                 <trust-store key-store-name="scarab"/>
                 <cipher-suite names="TLS_AES_128_GCM_SHA256"/>
             </ssl-context>
+            <ssl-context name="server-only-ssl-tls13">
```

Comment:
To make it clear for someone looking at this config later on, I think it would be good to rename this to something like "client-tls12-only".

Reply:
I also made the same change for the tests themselves, so they're now `testServerTLS12Only()` and `testClientOnlyTLS12Only()`